### PR TITLE
WormBase ParaSite protein trees parameter tweaks

### PR DESCRIPTION
**Pull request by fayerodgers**
## Description

Changes to parameters used for the WormBase ParaSite run of the protein trees pipeline. We used these parameters for WBPS release 16.

### CODE REVIEW

File: modules/Bio/EnsEMBL/Compara/PipeConfig/EG/WBParaSiteProteinTrees_conf.pm [hunk 1]
```diff
@@ -1,7 +1,7 @@
 =head1 LICENSE
 
-See the NOTICE file distributed with this work for additional information
-regarding copyright ownership.
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
```
Comment (JAlvarezJarreta) [inline modules/Bio/EnsEMBL/Compara/PipeConfig/EG/WBParaSiteProteinTrees_conf.pm:4]: Our header we have follows the latest Ensembl Software License policy (and avoids us to have to change it every year), so please revert this change.

**Review by JAlvarezJarreta — CHANGES_REQUESTED** [review]

File: modules/Bio/EnsEMBL/Compara/PipeConfig/EG/WBParaSiteProteinTrees_conf.pm (hunk 1 above)
Comment (fayerodgers) [reply modules/Bio/EnsEMBL/Compara/PipeConfig/EG/WBParaSiteProteinTrees_conf.pm]: Done, thanks :)

**Review by JAlvarezJarreta — APPROVED** [review]
